### ISSUE TITLE
ES6 imports as entry point?

### ISSUE THREAD

**Issue by timojuonoja** [highcharts/highcharts-react#64]
Jest fails with:
```
highcharts-react-official/src/HighchartsReact.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import React from "react"

SyntaxError: Unexpected token import
```

Why won't you provide precompiled CommonJS?

**Comment by ppotaczek** [comment]
Hi @timojuonoja,

Thank you for pointing out this important problem. I will soon add the corrections.

**Comment by waqas19921** [comment]
Quick fix for this is copy HighchartsReact.js to your project and import into your component.

**Comment by timojuonoja** [comment]
@waqas19921 Did that already, but good that you mentioned so that some other will do the same for now (once this is fixed).

**Comment by Kepro** [comment]
oh... this is a painful issue :(

**Comment by tobias-zucali** [comment]
My quick fix was to downgrade to version 1.4.0. Hope this issue will be fixed soon!

**Comment by ppotaczek** [comment]
The problem has been fixed. Now you can use the new version of the wrapper - 2.0.0 with precompiled code.
